gallery/src/components/NxMultiFileUpload/NxMultiFileUploadPage.tsx

```diff
@@ -0,0 +1,219 @@
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxMultiFileUploadExample from './NxMultiFileUploadExample';
+import NxMultiFileUploadComplexExample from './NxMultiFileUploadComplexExample';
+import NxMultiFileUploadDisabledExample from './NxMultiFileUploadDisabledExample';
+
+const nxMultiFileUploadCode = require('./NxMultiFileUploadExample?raw');
+const nxMultiFileUploadComplexCode = require('./NxMultiFileUploadComplexExample?raw');
+const nxMultiFileUploadDisabledCode = require('./NxMultiFileUploadDisabledExample?raw');
+
+const NxTextInputPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxCode>NxMultiFileUpload</NxCode> is an RSC user interface for a file-selection form field
+        (an <NxCode>&lt;input type="file"&gt;</NxCode>). Multiple files can be selected with multiple calls
+        to the <NxCode>onChange</NxCode> prop. Any selected files are displayed with the name and size of the
+        file, along with a button to remove the selection.
+      </NxP>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>Props</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Name</NxTable.Cell>
+              <NxTable.Cell>Type</NxTable.Cell>
+              <NxTable.Cell>Required</NxTable.Cell>
+              <NxTable.Cell>Default</NxTable.Cell>
+              <NxTable.Cell>Description</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell>files</NxTable.Cell>
+              <NxTable.Cell>FileList | null</NxTable.Cell>
+              <NxTable.Cell>true</NxTable.Cell>
+              <NxTable.Cell/>
+              <NxTable.Cell>
+                The{' '}
+                <NxCode>
+                  <NxTextLink href="https://developer.mozilla.org/en-US/docs/Web/API/FileList" external>
+                    FileList
+                  </NxTextLink>
+                </NxCode>
+                {' '}representing the user's current file selections, if any. Typically will be null before
+                the user makes a selection.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>onChange</NxTable.Cell>
+              <NxTable.Cell>(FileList | null) =&gt; void</NxTable.Cell>
+              <NxTable.Cell>true</NxTable.Cell>
+              <NxTable.Cell/>
+              <NxTable.Cell>
+                A callback which is called whenever the user updates their file selections. The new selection
+                {' '}/ selections, in the form of a <NxCode>FileList</NxCode>, is passed as the parameter, and
+                added to any existing file selections. If the user has removed all of their selections, the
```

Comment:
What does "and added to any existing file selections" mean here? Doesn't the `FileList` passed to this callback represent the _entire_ current selection, including both old and newly selected items?

Reply:
> What does "and added to any existing file selections" mean here? Doesn't the `FileList` passed to this callback represent the _entire_ current selection, including both old and newly selected items?

Yep you're right, it was written unclearly. Changed here: [8a3054c](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/8a3054ce1818850deb00183b7c0f9d38fe861437)